### PR TITLE
represent unsignedLong as string so that we don't get a 53 bit limitation

### PR DESCRIPTION
**Pull request by rolfyone**
represent unsignedLong as string so that we don't get a 53 bit limitation applied in some JS clients.

Signed-off-by: Paul Harris <paul.harris@consensys.net>

### CODE REVIEW

File: data/provider/src/main/java/tech/pegasys/artemis/provider/UnsignedLongSerializer.java [hunk 1]
```diff
@@ -23,6 +23,6 @@
   @Override
   public void serialize(UnsignedLong value, JsonGenerator gen, SerializerProvider serializers)
       throws IOException {
-    gen.writeNumber(value.bigIntegerValue());
+    gen.writeString(value.toString(10));
```
Comment (macfarla) [inline data/provider/src/main/java/tech/pegasys/artemis/provider/UnsignedLongSerializer.java:26]: does 10 here mean base 10?

**Review by macfarla — APPROVED** [review]
LGTM - do we need a test to show the 53 bit case doesn't break it?

File: data/provider/src/main/java/tech/pegasys/artemis/provider/UnsignedLongSerializer.java (hunk 1 above)
Comment (rolfyone) [reply data/provider/src/main/java/tech/pegasys/artemis/provider/UnsignedLongSerializer.java:26]: yep

**Review by macfarla — APPROVED** [review]
LGTM

**Pull request merged**